guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -230,7 +232,7 @@ Type names must be enclosed in curly braces. If the type is omitted, the compile
 };
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-return">@return</h4>
+#### @return
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-return}` to preserve the custom id.